### PR TITLE
IBX-8916: Updates in User documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-8917)
| Versions      | 4.6+

Updates in User documentation

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/customer_management/customer_management.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+description: Customer management allows you to manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.
+---
+
+# Customer management
+
+Customer management allows you to manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.
```
Comment (juskora) [inline docs/customer_management/customer_management.md:7]:
```suggestion
Customer management allows you to manage and view accounts for customers and organizations in your system, as well as create customer portals and allows for company self-registration.
```

Comment (dabrt) [inline docs/customer_management/customer_management.md:7]:
```suggestion
By using customer management features you can manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.
```

File: docs/customer_management/customer_management.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+description: Customer management allows you to manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.
```
Comment (dabrt) [inline docs/customer_management/customer_management.md]:
```suggestion
description: Use customer management features to manage and view accounts for customers and organizations in your system, as well as create customer portals and allow for company self-registration.
```

File: docs/getting_started/dashboard/dashboard.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ description: Get to know about dashboard.
 Dashboard is the first screen that you see after you log into [[= product_name =]].
 It provides users a quick overview about specific content to make better business decisions based on data and work more efficiently with a clear and easy access to important information.
 
-Dashboard in [[= product_name_headless =]] contains the following blocks:
+Default dashboard in [[= product_name_headless =]] contains the following blocks:
```
Comment (dabrt) [inline docs/getting_started/dashboard/dashboard.md:10]:
```suggestion
The default dashboard in [[= product_name_headless =]] contains the following blocks:
```

File: docs/getting_started/dashboard/work_with_dashboard.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ To do it, click the **Switch layout** icon on the left side of the upper toolbar
 
 ### Add blocks
 
-In Dashboard Builder, you can access a menu of **Elements**, which includes a set of blocks of content that you can add to the zones of the dashboard.
+In Dashboard Builder, you can access a menu of **Dashboard blocks**, which includes a set of blocks of content that you can add to the zones of the dashboard.
```
Comment (dabrt) [inline docs/getting_started/dashboard/work_with_dashboard.md:74]:
```suggestion
In Dashboard Builder, you can access the **Dashboard blocks** menu, which includes a set of blocks of content that you can add to the zones of the dashboard.
```

File: docs/getting_started/dashboard/work_with_dashboard.md [hunk 5]
```diff
@@ -84,14 +84,14 @@ You can reorder blocks in a few ways:
 
 - drag and drop block in the desired location on a drop zone
 - use **Move up block** or **Move down block** option from the block toolbar
-- access Structure view and use 'Move up' and 'Move down' function in the settings of the block
+- access Structure view and use **Move up** or **Move down** option in the settings of the block
```
Comment (dabrt) [inline docs/getting_started/dashboard/work_with_dashboard.md:87]:
```suggestion
- access the **Structure view** menu and use **Move up** or **Move down** option in the settings of the block
```

File: docs/getting_started/dashboard/work_with_dashboard.md [hunk 6]
```diff
@@ -47,11 +47,11 @@ C. Buttons:
 |Redo|![Redo](dashboard_redo.png)|Redo latest change.|
 |Structure view|![Structure view](dashboard_structure_view.png)|Access Structure view toolbar.|
 |Dashboard blocks|![Dashboard blocks](dashboard_blocks.png)|Access Dashboard blocks toolbar.|
-|View switch|![View switch](dashboard_switch_toolbar.png)|Move Elements / Structure toolbar to the other side of the screen.|
+|View switch|![View switch](dashboard_switch_toolbar.png)|Move toolbar to the other side of the screen.|
```
Comment (dabrt) [inline docs/getting_started/dashboard/work_with_dashboard.md:50]:
```suggestion
|View switch|![View switch](dashboard_switch_toolbar.png)|Move menu to the other side of the screen.|
```
I am wondering where the "View switch" name comes from. The name does not really reflect its function. (Not a review comment)

File: docs/getting_started/dashboard/work_with_dashboard.md [hunk 7]
```diff
@@ -47,11 +47,11 @@ C. Buttons:
 |Redo|![Redo](dashboard_redo.png)|Redo latest change.|
 |Structure view|![Structure view](dashboard_structure_view.png)|Access Structure view toolbar.|
 |Dashboard blocks|![Dashboard blocks](dashboard_blocks.png)|Access Dashboard blocks toolbar.|
```
Comment (dabrt) [inline docs/getting_started/dashboard/work_with_dashboard.md:49]:
```suggestion
|Structure view|![Structure view](dashboard_structure_view.png)|Access **Structure view** menu.|
|Dashboard blocks|![Dashboard blocks](dashboard_blocks.png)|Access **Dashboard blocks** menu.|
```

Also, in line 38 above, please change "toolbars" to "menus"

File: docs/getting_started/dashboard/work_with_dashboard.md [hunk 8]
```diff
@@ -47,11 +47,11 @@ C. Buttons:
...
-|View switch|![View switch](dashboard_switch_toolbar.png)|Move Elements / Structure toolbar to the other side of the screen.|
+|View switch|![View switch](dashboard_switch_toolbar.png)|Move toolbar to the other side of the screen.|
 
 Dashboard Builder has two main toolbars that you can use while creating a dashboard:
```
Comment (dabrt) [inline docs/getting_started/dashboard/work_with_dashboard.md:52]:
```suggestion
Dashboard Builder has two main menus that you can use while creating a dashboard:
```

File: docs/content_management/content_management.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
+---
+description: Content management enables you to create, edit, and manage content items and content types. It also includes content translation and publication scheduling.
+---
+
+# Content management
+
+Content management enables you to create, edit, and manage content items and content types. It also includes content translation and publication scheduling.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/content_management.md:7]: It's a landing page. I don't see a point of inserting link to content types here.

[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/content_management.md:7]: + I like the first sentence more.

File: docs/content_management/content_management.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
+---
+description: Content management enables you to create, edit, and manage content items and content types. It also includes content translation and publication scheduling.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/content_management.md:2]: Two comments for the same line.

File: docs/customer_management/customer_management.md (hunk 1 above)
Comment (julitafalcondusza) [reply docs/customer_management/customer_management.md:7]: I'd keep first version.

File: docs/getting_started/dashboard/work_with_dashboard.md (hunk 6 above)
Comment (julitafalcondusza) [reply docs/getting_started/dashboard/work_with_dashboard.md:50]: Following Ibexa Design System - this element is called 'toolbox', so I'll update it.

File: docs/getting_started/dashboard/work_with_dashboard.md (hunk 4 above)
Comment (julitafalcondusza) [reply docs/getting_started/dashboard/work_with_dashboard.md:74]: Updated with 'toolbox'.

File: docs/getting_started/dashboard/work_with_dashboard.md (hunk 7 above)
Comment (julitafalcondusza) [reply docs/getting_started/dashboard/work_with_dashboard.md:49]: Updated with 'toolbox'.

File: docs/getting_started/dashboard/work_with_dashboard.md (hunk 8 above)
Comment (julitafalcondusza) [reply docs/getting_started/dashboard/work_with_dashboard.md:52]: Toolboxes.

**Pull request merged**